crowd_sim/envs/crowd_sim.py

```diff
@@ -149,8 +190,14 @@ def generate_random_human_position(self, human_num, rule):
                     else:
                         human = self.generate_square_crossing_human()
                     self.humans.append(human)
+        elif rule == 'static': ## Only test for generating static obstacles 
```

Comment:
I think the naming is a little bit misleading. Can we rename it as follows (or something else you would suggest)
```suggestion
        elif rule == 'test': ## Only test for generating static obstacles 
```

Reply:
Yes! we can!